core/templates/dev/head/components/HintEditorDirective.js

```diff
@@ -60,6 +61,18 @@ oppia.directive('hintEditor', [
             $scope.hintEditorIsOpen = false;
           };
 
+          $scope.onAudioTranslationsStartEditAction = function() {
```

Comment:
OK to postpone to the future (file an issue if so?) but do you think it would make sense to have a standard SubtitledHtmlEditorDirective that has capability for editing subtitled-html-with-audio? I'm just noticing the same code repeated all over the place.

Abstraction is not always a positive, so this is actually a legitimate question -- maybe there's a good reason for not doing a reusable thing here. (But, if so, what is it?)

Reply:
Yes, I think that sounds reasonable. I'll file an issue for it after this PR gets in.